src/index.ts

```diff
@@ -1,48 +1,55 @@
-import { app, ipcMain } from 'electron';
-import { appManager } from './main/AppManager';
-import { TrayMenu } from './main/TrayMenu';
-import { AppWindow } from './main/AppWindow';
-import { ObserverManager } from './main/observers/ObserverManager';
+import { app, ipcMain } from "electron";
+import { appManager } from "./main/AppManager";
+import { TrayMenu } from "./main/TrayMenu";
+import { AppWindow } from "./main/AppWindow";
+import { ObserverManager } from "./main/observers/ObserverManager";
 import "./store";
-import { NotificationManager } from './main/NotificationManager';
-import { dirname, resolve, basename } from 'path';
+import { NotificationManager } from "./main/NotificationManager";
+import { dirname, resolve, basename } from "path";
+import updateElectronApp from "update-electron-app";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import hasSquirrelStartupEvents from "electron-squirrel-startup";
```

Comment:
@pehota where is this on the package.json

Reply:
@kanekotic ,
Those had already been there. I only reformatted the import style. I believe those come with electron.